README.md

```diff
@@ -2,7 +2,9 @@
 
 A nova field for Laraberg
 
-## Installation
+## Quick start
+
+### Installation
```

Comment:
Zou het goed zijn ook in de readme aan te geven wat de minimale requirements zijn? Bijvoorbeeld omdat de PHP versie nu naar `8.1` is gegaan?

Reply:
The PHP version is set in the composer.json file. I personally think this is clean enough, as a user I would expect this to follow Nova's minimum requirements, and PHP's supported versions.
Right now Nova is laravel8/PHP7.3, minimum supported PHP version is 8.1, so PHP 8.1?

Reply:
You often see the versions in readme's. Even though a version is configured in `composer.json` and the package relies on Nova, it wouldn't hurt to put it in the readme, so anyone that wants to use the package can immediately see what the minimum requirements are.